globals.h

```diff
@@ -54,5 +54,9 @@ extern int white_moves[60];
    but all updates must be reversed when the search stops. */
 extern Board board;
 
+#ifdef ANDROID
+int droidzebra_message_debug(const char* format, ...);
```

Comment:
not the nicest thing in the world but what can we do 🤷‍♂️ 

Ideally this project should not depend on droidzebra like this, so the function should be somehow injected from the outside, but we got no time for that business

Reply:
maybe this could just be a standard debug function like message_debug ...